src/main/java/com/cglx/Calculator.java

```diff
@@ -37,25 +37,10 @@ public int square(int a)
         return a * a;
     }
 
-    private List<String> readFile(String filename)
+    public int square(int a)
     {
-      List<String> records = new ArrayList<String>();
-      try
-      {
-        BufferedReader reader = new BufferedReader(new FileReader(filename));
-        String line;
-        while ((line = reader.readLine()) != null)
-        {
-          records.add(line);
-        }
-        return records;
-      }
-      catch (Exception e)
-      {
-        //System.err.format("Exception occurred trying to read '%s'.", filename);
-        e.printStackTrace();
-        return null;
-      }
+        int result = a * a;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AS1488)